### ISSUE TITLE
Import Using SiteIDs from File Option fails for 15 digit site IDs when using site list retreived from QWDATA

### ISSUE THREAD

**Issue by alfred-usgs** [USGS-R/QWToolbox#78]
It's convenient to use a site list retrieved from QWDATA when reviewing data in QWToolbox.  They work fine for 8 digit site IDs, but fail for sites with 15-digit IDs.  I suspect it has to do with no space/ delimiter between the 15th digit of the Site ID and the first character of the site name. 

 A qwdata site list of mixed 8 and 15 digit site ID's will import data for the 8 digit IDs only, and skip the 15 digit IDs.

![image](https://cloud.githubusercontent.com/assets/10554968/6093556/a45adede-aeb8-11e4-9d7e-7de864e1666c.png)

**Comment by mcanova-usgs** [comment]
I tried a mix of 8 and 15 digit site ids as formatted below and it did work.

08068500       
08070200       
295554095093401
295826095082200

**Comment by tmills-usgs** [comment]
You can do a mix of site IDs, but the "USGS" header is not allowed .This has come to my attention recently as an enhancement. I will put a note down to allow for using a QWData site list directly out of QWData. I am surprised that file worked at all, it should not have. Can you send me it?

**Comment by alfred-usgs** [comment]
Yes, it works with 15 digit IDs when there is nothing after the end of the ID. Can you try using a site file saved from a QWDATA retrieval? The site name field ends up right against the end of the ID, and the 15-digit IDs are skipped on import.

**Comment by alfred-usgs** [comment]
Hi Joe,

I've attached 2 files.  QWDATA.sites is the the standard output file from
QWDATA that results from the "do you want to save a list of sites" prompt
after selecting records in the output menu. The second file "edited.QWDATA"
works (I deleted the station name from the 15-digit ID sites).
BTW, I love this interface.  Being able to jump through so many plots for
multiple sites is absolutely AWESOME!

Amanda

On Mon, Feb 9, 2015 at 5:52 AM, T. Joe Mills <notifications@github.com>
wrote:

> You can do a mix of site IDs, but the "USGS" header is not allowed .This
> has come to my attention recently as an enhancement. I will put a note down
> to allow for using a QWData site list directly out of QWData. I am
> surprised that file worked at all, it should not have. Can you send me it?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/USGS-R/QWToolbox/issues/78#issuecomment-73511279>.
>



-- 
Amanda L. Fredericksen, Hydrologist
U.S. Geological Survey
California Water Science Center
------------------------------------------------------
6000 J Street, Placer Hall
Sacramento, CA 95819
alfred@usgs.gov
Office- 916-278-3210
Mobile- 317-407-7550
http://ca.water.usgs.gov/ <http://in.water.usgs.gov/>
